docs/README.md

```diff
@@ -82,28 +82,28 @@ const bot = mineflayer.createBot({
   port: 25565,       // optional
   username: 'email@example.com', // email and password are required only for
   password: '12345678',          // online-mode=true servers
-  version: false,                 // false corresponds to auto version detection (that's the default), put for example "1.8.8" if you need a specific version
-  auth: 'mojang'      // optional; by default uses mojang, if using a microsoft account, set to 'microsoft'
+  // version: false,             /* only set if you need a specific version or snapshot (ie: "1.8.9" or "1.16.5"), otherwise it's set automatically */
+  // auth: 'mojang'              /* only set if you need microsoft auth, then set this to 'microsoft' */
```

Comment:
This isn't really consistent - Why comment out optional properties here and use multi-line comments for commenting at the end of lines, when other optional properties are not commented out and the regular single-line comment format is used everywhere?

Reply:
fixed.